Service/Microsoft.Health.DeIdentification.Fhir/FhirDeIdOperation.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Health.DeIdentification.Contract;
+using Microsoft.Health.Fhir.Anonymizer.Core;
+using Microsoft.Health.Fhir.Anonymizer.Core.AnonymizerConfigurations;
+using Microsoft.Health.Fhir.Anonymizer.Core.PartitionedExecution;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Health.DeIdentification.Fhir
+{
+    public class FhirDeIdOperation : IDeIdOperation<List<Object>, string>
+    {
+        private AnonymizerEngine _anonymizerEngine;
+
+        public FhirDeIdOperation(string configContext)
+        {
+            AnonymizerEngine.InitializeFhirPathExtensionSymbols();
+            _anonymizerEngine = new AnonymizerEngine(AnonymizerConfigurationManager.CreateFromSettingsInJson(configContext));
+        }
+
+        public string Process(List<Object> source)
+        {
+            var result = new StringBuilder();
+            foreach (var item in source)
+            {
+                result.Append(ProcessSingle(item.ToString()));
+            }
+            return result.ToString();
```

Comment:
So this operation deids a resource list and generates a result string? How we determine the delimiters for each input in the result string?

Reply:
Now I just use stringbuilder to generate output string and maybe I can use AppendLine to delimit result.

Reply:
I think maybe we need to keep the same format as the input.

Reply:
Ok, I will change the output type to List